dissect/target/filesystem.py

```diff
@@ -1115,13 +1131,18 @@ def map_dir(self, vfspath: str, realpath: str) -> None:
                 real_file_path = os.path.join(root, file_)
                 directory.add(file_, MappedFile(self, vfs_file_path, real_file_path))
 
-    def map_file(self, vfspath: str, realpath: str) -> None:
+    def map_file(self, vfspath: str, realpath: str, compression: str = "") -> None:
```

Comment:
Very nitpicky once again, but would a default of `compression = None` not make more sense?

Reply:
Hmm, that should either read "gzip"  or not be a keyword arg. With `None` or `""` the default constructor will always raise a `ValueError`, which is not correct API behaviour.

Reply:
Ah wait, this is `map_file()`, nvm, I was thinking about the `__init__()` of `MappedCompressedFile`. None would be nicer here as a signaling value.